packages/backend/src/accounting/service.ts

```diff
@@ -80,7 +81,9 @@ export interface AccountingService {
   createSettlementAccount(unit: number): Promise<void>
   getBalance(id: string): Promise<bigint | undefined>
   getTotalSent(id: string): Promise<bigint | undefined>
-  getTotalReceived(id: string): Promise<bigint | undefined>
+  getAccountsTotalSent(ids: string[]): Promise<(bigint | undefined)[]>
+  getAccountTotalReceived(id: string): Promise<bigint | undefined>
```

Comment:
Either `getTotalSent` --> `getAccountTotalSent`
or `getAccountTotalReceived` --> `getTotalReceived`

Reply:
So weird. I could have sworn that I didn't change it from `getTotalReceived` to `getAccountTotalReceived`.